app-web/src/pages/Retrospection/RetrospectionModal.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from '@emotion/styled';
+import { Button, Dialog, IconButton, TextField } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import { useState } from 'react';
+
+const Wrap = styled.div`
+  padding:3rem;
+  display: flex;
+  flex-direction: column;
+`;
+
+const ButtonWrap = styled.div`
+  display: flex;
+  justify-content: flex-end;
+
+  button:nth-child(1){
+      margin-right:1rem
+  }
```

Comment:
```suggestion
  button:nth-child(1) {
    margin-right: 1rem;
  }
```
정렬이 잘 되어야 할 것 같아요. VS Code의 확장 프로그램을 설치하면 실수를 줄일 수 있을거예요

See also
  - https://marketplace.visualstudio.com/items?itemName=blanu.vscode-styled-jsx